pkg/apis/experimental/types.go

```diff
@@ -639,3 +639,36 @@ type ClusterAutoscalerList struct {
 
 	Items []ClusterAutoscaler `json:"items"`
 }
+
+// Selector is a label query over a set of API objects. MatchLabels and
+// MatchExpressions are ANDed. An empty Selector matches all objects. A
+// null Selector matches no objects.
+type Selector struct {
+	// MatchLabels is a list of labels.
```

Comment:
I'm not convinced those are sufficient justifications. Copying the existing label selector syntax into the MatchLabels field is only marginally easier than rewriting it into the MatchExpressions syntax. Having both makes this structure unnecessarily confusing long-term IMO.